Cython/Compiler/ExprNodes.py

```diff
@@ -7216,6 +7217,7 @@ class AttributeNode(ExprNode):
     #  is_called            boolean   Function call is being done on result
     #  entry                Entry     Symbol table entry of attribute
 
+    child_attrs = ["attribute"]
```

Comment:
This definitely doesn't fit our `child_attrs` scheme - `attribute` is a string, but the `child_attrs` should inherit from `Node` (or be lists of things that inherit from `Node`)